package.json

```diff
@@ -1,52 +1,52 @@
 {
-    "name": "kinvey-nativescript-sdk",
-    "version": "3.8.0",
-    "description": "Kinvey JavaScript SDK for NativeScript applications.",
-    "homepage": "http://www.kinvey.com",
-    "bugs": {
-        "url": "https://github.com/Kinvey/nativescript-sdk/issues"
-    },
-    "license": "Apache-2.0",
-    "author": "Kinvey",
-    "contributors": [
-        "Thomas Conner <thomas@kinvey.com>"
-    ],
-    "main": "./src/index.js",
-    "typings": "./src/kinvey.d.ts",
-    "repository": {
-        "type": "git",
-        "url": "git@github.com:Kinvey/nativescript-sdk"
-    },
-    "scripts": {
-        "build": "rm -rf src/**/*.js && rm -rf push/**/*.js && npm i && tsc",
-        "ci.tslint": "npm i && tslint **/*.ts",
-        "lint": "tslint **/*.ts",
-        "preversion": "rm -rf node_modules && npm install && npm run build",
-        "postversion": "git push && git push --tags"
-    },
-    "dependencies": {
-        "es6-promise": "4.1.0",
-        "events": "1.1.1",
-        "kinvey-js-sdk": "3.8.0",
-        "nativescript-sqlite": "1.1.7"
-    },
-    "devDependencies": {
-        "nativescript-dev-typescript": "0.4.5",
-        "tns-core-modules": "3.0.1",
-        "tns-platform-declarations": "3.0.1",
-        "tslint": "^5.4.3",
-        "tslint-config-airbnb": "5.3.0",
-        "typescript": "2.3.4"
-    },
-    "nativescript": {
-        "platforms": {
-            "ios": "3.0.0",
-            "android": "3.0.0"
-        }
-    },
-    "keywords": [
-        "Kinvey",
-        "JavaScript",
-        "NativeScript"
-    ]
+  "name": "kinvey-nativescript-sdk",
+  "version": "3.8.0",
+  "description": "Kinvey JavaScript SDK for NativeScript applications.",
+  "homepage": "http://www.kinvey.com",
+  "bugs": {
+    "url": "https://github.com/Kinvey/nativescript-sdk/issues"
+  },
+  "license": "Apache-2.0",
+  "author": "Kinvey",
+  "contributors": [
+    "Thomas Conner <thomas@kinvey.com>"
+  ],
+  "main": "./src/index.js",
+  "typings": "./src/kinvey.d.ts",
+  "repository": {
+    "type": "git",
+    "url": "git@github.com:Kinvey/nativescript-sdk"
+  },
+  "scripts": {
+    "build": "rm -rf src/**/*.js && rm -rf push/**/*.js && npm i && tsc",
+    "ci.tslint": "npm i && npm run lint",
+    "lint": "tslint **/*.ts --exclude **/node_modules/**/* --exclude **/*.d.ts",
```

Comment:
@dimofeev I thought I had read that you could use `tslint.json` to exclude files but now I am unable to find it. This looks good.